src/graccreq/oim/OIMTopology.py

```diff
@@ -104,7 +110,7 @@ def get_file_from_OIM():
         try:
             oim_xml = urllib2.urlopen(oim_url)
         except (urllib2.HTTPError, urllib2.URLError) as e:
-            print e
+            logging.warn(e)
```

Comment:
I suppose it depends on the logic.  It this fails, but it just falls back to the older version from OIM, then sure, warning.  But if it fails, then just bails and doesn't fill in the records, it should be a warning.